app/dashboard/views.py

```diff
@@ -1141,6 +1151,188 @@ def get_quickstart_video(request):
+
+    # Header
+    d_expiring_bounties_count = expiring_bounties.count()
+    d_expiring_days_count = get_expiring_days_count(expiring_bounties)
+
+    current_funder_bounties = funder_bounties.filter(current_bounty=True)
+
+    # Stats
+    d_submitted_bounties_count = current_funder_bounties.count()
+    d_total_contributors_count = 0
+    for bounty in current_funder_bounties:
+        d_total_contributors_count += bounty.fulfillments.filter(accepted=True).count()
+
+    d_top_contributors = get_top_contributors(done_bounties_desc_created, 12)
+
+    d_total_paid_dollars = 0
+    d_total_paid_eth = 0
+
+    for bounty in done_bounties:
+        bounty_value_in_usdt = bounty.get_value_in_usdt
+        bounty_value_in_eth = eth_from_wei(bounty.get_value_in_eth)
+
+        if bounty_value_in_usdt is not None:
+            d_total_paid_dollars = float(d_total_paid_dollars) + float(bounty.get_value_in_usdt)
+
+        d_total_paid_eth = float(d_total_paid_eth) + float(bounty_value_in_eth)
+
+    paid_date_since = None
+    if done_bounties_desc_created.last() is not None:
+        paid_date_since = done_bounties_desc_created.last().web3_created
+
+    d_total_paid_date_since = None
+    if paid_date_since is not None:
+        d_total_paid_date_since = paid_date_since.strftime('%d %b, %y')
+
+    # total budget
+    total_budget_updated_on = request.user.profile.funder_total_budget_updated_on
+    budget_type = request.user.profile.funder_total_budget_type
+
+    d_total_budget_use_input_layout = is_funder_allowed_to_input_total_budget(total_budget_updated_on, budget_type)
+
+    funder_total_budget = request.user.profile.funder_total_budget_usdt
+    total_budget_data = get_funder_total_budget(d_total_budget_use_input_layout, funder_total_budget, budget_type)
+
+    d_total_budget_dollars = total_budget_data['total_budget_dollars']
+    d_total_budget_eth = total_budget_data['total_budget_eth']
+    d_total_budget_used_time_period = total_budget_data['total_budget_used_time_period']
+
+    # Latest on your bounties
+    utc_now = datetime.datetime.now(timezone.utc)
+    d_tax_year = utc_now.year
+    d_tax_year_bounties_count = 0
+    d_tax_year_bounties_worth_dollars = 0
+
+    for bounty in done_bounties:
+        accepted_on = bounty.fulfillment_accepted_on
+        if accepted_on is not None and d_tax_year == accepted_on.year and bounty.get_value_in_usdt:
+            d_tax_year_bounties_count += 1
+            d_tax_year_bounties_worth_dollars += float(bounty.get_value_in_usdt)
+
+    d_expired_issues_count = expired_bounties.count()
+    d_expired_issues_worth_dollars = 0
+    for expired_issue in expired_bounties:
+        issue_worth_in_usdt = expired_issue.get_value_in_usdt
+        if issue_worth_in_usdt is not None:
+            d_expired_issues_worth_dollars = float(d_expired_issues_worth_dollars) + float(issue_worth_in_usdt)
+
+    d_active_bounties_count = active_bounties.count()
```

Comment:
If we're using a variable in only one place, can we omit the variable assignment and use whatever the variable is calling in line?  In this case, `active_bounties.count()` could be on line 1292